packages/admin-panel/src/authentication/actions.js

```diff
@@ -63,3 +64,17 @@ export const loginError = errorMessage => ({
 export const logout = () => ({
   type: LOGOUT,
 });
+
+// Profile
+export const updateProfile = (id, payload) => async (dispatch, getState, { api }) => {
+  try {
+    await api.put(`user/${id}`, null, payload);
```

Comment:
We should be able to use a `/me` endpoint here, which can then have built in permission checking that the same user is making the request as is being edited